README.md

```diff
@@ -8,6 +8,15 @@ We hope you enjoy using the Microsoft JDBC Driver for SQL Server.
 
 SQL Server Team
 
+## Status of Most Recent Builds
+| AppVeyor (Windows)       | Travis CI (Linux) |
+|--------------------------|--------------------------|
+| [![av-image][]][av-site] | [![tv-image][]][tv-site] |
+
+[av-image]: https://ci.appveyor.com/api/projects/status/o6fjg16678ol64d3?svg=true
```

Comment:
You can use optional title after your svg image like `?svg=true "Optional Title"` which will mimic effect of onhover link. 

| [![av-image][]][av-site] | [![tv-image][]][tv-site] |

[av-image]: https://ci.appveyor.com/api/projects/status/o6fjg16678ol64d3?svg=true "Windows"
[av-site]: https://ci.appveyor.com/project/Microsoft-JDBC/mssql-jdbc
[tv-image]: https://travis-ci.org/Microsoft/mssql-jdbc.svg? "Linux"
[tv-site]: https://travis-ci.org/Microsoft/mssql-jdbc

Still I like idea of all badges on top of README.md just like  https://github.com/travis-ci/travis-api